third-party/chpl-venv/README.md

```diff
@@ -76,6 +80,14 @@ A YAML parser and emitter for Python
 
 **Website**: https://pypi.python.org/pypi/PyYAML
 
+### scspell
+
+A spell checker for source code
+
+**License**: scspell program: GPL 2; wordlists: public domain / assorted: https://github.com/en-wl/wordlist/blob/master/scowl/Copyright
```

Comment:
Package dependencies for scspell should be listed in the Package Dependencies section below (with a note as to which package requires them)